src/test/codegen/sanitizer-recover.rs

```diff
@@ -1,6 +1,7 @@
 // Verifies that AddressSanitizer and MemorySanitizer
 // recovery mode can be enabled with -Zsanitizer-recover.
 //
+// no-system-llvm
```

Comment:
In your diff this one fails with `failed to get bitcode from object file for LTO (Bitcode section not found in object file)`. That seems like an unrelated issue?

Reply:
I agree it seems unrelated, but the test stops failing with the alloc patch applied. It's as mysterious to me as it sounds like it is to you.